docs/configuration.md

```diff
@@ -206,6 +206,19 @@ Any flag that has a value that is a comma separated list on the command line bec
     poll-interval = "0m15s"
     ```
 
+##### Metric Diagnostics
+
+* Description: Enable diagnostic reporting. To disable diagnostics set to false.
+* Flag: `metric.diagnostics`
+* Env: `PILOSA_METRIC_DIAGNOSTICS`
+* Config:
+
+    ```toml
+    [metric]
+    diagnostics = true
```

Comment:
Seems that `false` is not a valid value for the falsy value. `0` seems to disable it. Would be good to notify user if the value is not valid.

Reply:
I tested this and false does seem to work.  I updated to docs with clearer instructions.